### ISSUE TITLE
usbmc.skprx and gamesd.skprx won't work together

### ISSUE THREAD

**Issue by milkawax** [yifanlu/usbmc#8]
I have both cards with sd in my 3G vita and want to load both drivers.
Is ther a way to load usbmc and sd2vita at same time,
One as ux0 and one as uma0 ? Or anything else ?
Yesterday read something about ima0 or something.
Maybe is possible to mount all three cards at once (ux0/uma0/ima0) ?
Want to make my Vita retro console.
:)

**Comment by yifanlu** [comment]
Yes that is a known issue. I have no time to do something like this, if someone does it and sends a pull request, I'd be happy to review it.

**Comment by milkawax** [comment]
I've seen a screenshot with vshell picture where all 3 cards mounted at once (ux0/uma0/ima0), don't know where it was so i think someone was workkng on it or has done it already....
(ux0/uma0/ima0)

Von meinem iPhone gesendet

**Comment by jmota** [comment]
Is this possible? To mount the Sony Memorycard and the SD2Vita? If so i’d love to learn how to make it happen on my end. 😊👍

**Comment by IDBurner** [comment]
@milkawax I think you are looking for https://github.com/M1lk4h0l1c/usbmc/releases

I do not believe it has been updated for the latest 3.65 releases
